### ISSUE TITLE
Error: No tag TODAY

### ISSUE THREAD

**Issue by wendyong92** [johannesjo/super-productivity#724]
### Steps to Reproduce

1. shutdown laptop the previous night
2. start up laptop
3. open super productivity app
4. error occurs

### Stacktrace
```
webpack:///src/app/features/tag/store/tag.reducer.ts:46:12
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1096:29)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
args (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1059:38)
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1105:35)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
source$.pipe(pluck (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:868:24)
_next (webpack:///node_modules/rxjs/_esm2015/internal/operators/map.js:29:34)
next (webpack:///node_modules/rxjs/_esm2015/internal/Subscriber.js:49:17)
super._subscribe (webpack:///node_modules/rxjs/_esm2015/internal/BehaviorSubject.js:14:23)
```

### Meta Info
META: SP6.1.0 Electron – en-US – Win32 – Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) superProductivity/6.1.0 Chrome/85.0.4183.121 Electron/10.1.5 Safari/537.36

### Actions Before Error
```
1607287885680: [Persistence] Save to DB 
1607287885682: [Persistence] Save to DB 
1607287885684: [Persistence] Save to DB 
1607287886658: [Task] Add time spent 
1607287886662: [Project] Update Work End 
1607287886664: [Tag] Update Work End for Tag 
1607287886673: [Persistence] Save to DB 
1607287886674: [Persistence] Save to DB 
1607287886675: [Persistence] Save to DB 
1607287887032: [Task] Toggle start 
1607287887039: [Persistence] Save to DB 
1607287887042: [Task] UnsetCurrentTask 
1607287894887: [Task] Update Task 
1607287894892: [Persistence] Save to DB 
1607287894918: [Task] Update Task 
1607287894920: [Persistence] Save to DB 
1607287897765: [Task] SetSelectedTask 
1607287905846: [Task] Update Task 
1607287905850: [Persistence] Save to DB 
1607287905871: [Task] Update Task 
1607287905874: [Persistence] Save to DB 
1607309390289: [Layout] Hide SideBar 
1607309395515: [SP_ALL] Load(import) all data 
1607309395522: [SP_ALL] All Data was loaded 
1607309395555: [WorkContext] Set Active Work Context 
1607309395603: [Task] SetSelectedTask 
1607309439341: [SP_ALL] Load(import) all data 
1607309439357: [SP_ALL] All Data was loaded 
1607309439407: [WorkContext] Set Active Work Context 
1607309439489: [Task] SetSelectedTask
```

**Comment by johannesjo** [comment]
Hey there! Thanks for reporting. This should be fixed since version 6.1.1. Could you try using the latest release?

**Comment by johannesjo** [comment]
Also duplicate of #702
